app/src/test/java/com/example/maciej/smim/GameUnitTest.kt

```diff
@@ -0,0 +1,73 @@
+package com.example.maciej.smim
+
+import org.junit.Assert.*
+import org.junit.Before
+import org.junit.runner.RunWith
+import org.junit.Test
+
+import junitparams.JUnitParamsRunner
+import junitparams.Parameters
+
+@RunWith(JUnitParamsRunner::class)
+class GameUnitTest {
+    private lateinit var game: Game
+
+    @Before
+    fun setUp() {
+        this.game = Game()
+    }
+
+    @Test
+    @Parameters(method = "test_updateScore_with_given_score_Parameters")
+    fun test_updateScore_with_given_score(startingScore: IntArray, player: Int, score: Int) {
```

Comment:
**Kotlin 101**: `IntArray` is **_not_** comparable to `Array<Int>` using `assertArrayEquals`